yb-voyager/cmd/analyzeSchema.go

```diff
@@ -1819,20 +1828,20 @@ func packAndSendAnalyzeSchemaPayload(status string) {
 	for _, issue := range schemaAnalysisReport.Issues {
 		issue.SqlStatement = ""  // Obfuscate sensitive information before sending to callhome cluster
 		issue.ObjectName = "XXX" // Redacting object name before sending
-		/*
-			Removing Reason and Suggestion completely for now as there can be sensitive information in some of the cases
-			so will enable it later with proper understanding
-			some of the examples -
-			Reason:
-				Stored generated columns are not supported. [columns]
-				Unsupported datatype - xml on [column]
-				Unsupported datatype - xid on [column]
-				Unsupported PG syntax - [error msg from parser]
-				Policy require roles to be created. [role names]
-			Suggestion:
-				Foreign Table issue mentions Server name to be created.
-		*/
-		issue.Reason = "XXX"
+		for _, sensitiveReason := range reasonsIncludingSensitiveInformation {
+			if strings.Contains(issue.Reason, sensitiveReason) {
+				switch sensitiveReason {
+				case UNSUPPORTED_DATATYPE, UNSUPPORTED_DATATYPE_LIVE_MIGRATION:
+					//e.g. Reason "Unsupported datatype - xml on column - data"
+					//sending only "Unsupported datatype - xml"
+					issue.Reason = strings.Split(issue.Reason, "on column -")[0]
+				default:
+					issue.Reason = sensitiveReason
+				}
+			}
+		}
```

Comment:
basically you are checking and modifying the issueReason which have sensitive information?

Reply:
right